sdk/include/opentelemetry/sdk/trace/span_data.h

```diff
@@ -254,11 +254,8 @@ class SpanData final : public Recordable
   }
 
   void AddEvent(nostd::string_view name,
-                opentelemetry::common::SystemTimestamp timestamp =
-                    opentelemetry::common::SystemTimestamp(std::chrono::system_clock::now()),
-                const opentelemetry::common::KeyValueIterable &attributes =
-                    opentelemetry::common::KeyValueIterableView<std::map<std::string, int>>(
-                        {})) noexcept override
+                opentelemetry::common::SystemTimestamp timestamp,
+                const opentelemetry::common::KeyValueIterable &attributes) noexcept override
```

Comment:
```suggestion
                opentelemetry::common::SystemTimestamp timestamp =
                    opentelemetry::common::SystemTimestamp(std::chrono::system_clock::now()),
                const opentelemetry::common::KeyValueIterable &attributes =
                    opentelemetry::common::KeyValueIterableView<std::map<std::string, int32_t>>) noexcept override
```

Reverting to original commit